docs/runbooks/LowReadyVirtOperatorsCount.md

```diff
@@ -0,0 +1,27 @@
+# LowReadyVirtOperatorsCount 
+
+## Meaning
+
+The virt-operator is the first k8s operator coming to alive in a KubeVirt cluster. With the primary responsible for the installation, live-update, live-upgrade of a KubeVirt cluster, it monitors the life-cycle of top-level controllers such as virt-controller, virt-handler, virt-launcher, etc. and manages their reconciliation. In addition, virt-operator is responsible for cluster-wide tasks such as certificate rotation and some infrastructure management, etc.
+
+Note that virt-operator is not directly responsible for virtual machines in the cluster, its temporal unavailability should not affect the custom workloads. 
+
+This alert is triggered when some virt operators are running but not in the `Ready` state in the past 5 minutes. The virt-operator deployment has a default replica of 2 pods.
```

Comment:
@yuhaohaoyu I think a comma is missing after "but" and "state".

Reply:
This seems to be resolved.